lib/iex/lib/iex/helpers.ex

```diff
@@ -1193,7 +1193,7 @@ defmodule IEx.Helpers do
   ## Examples
 
       iex> pid("0.21.32")
-      #PID<0.21.32>
+      ~PID<0.21.32>
 
   """
   def pid(string) when is_binary(string) do
```

Comment:
Please add a TODO to pid/1 - port/1 - reference/1 as they can now be deprecated. :)